webapp/channels/src/packages/mattermost-redux/src/reducers/entities/channels.ts

```diff
@@ -241,7 +241,7 @@ function channels(state: IDMappedObjects<Channel> = {}, action: GenericAction) {
     }
 }
 
-function toClientChannel(serverChannel: ServerChannel): Channel {
+export function toClientChannel(serverChannel: ServerChannel): Channel {
```

Comment:
This change is not needed anymore, right?

Reply:
Done.